gitpod-io/gitpod
# [code] Gitpod walkthrough

## Description


Update latest VS Code to bring Gitpod walkthrough: https://github.com/gitpod-io/openvscode-server/pull/110

Gitpod Code commit: https://github.com/gitpod-io/openvscode-server/commit/128bab190731bc4af1a6e58b9324110a0d087884

## Related Issue(s)

Related to https://github.com/gitpod-io/gitpod/issues/5964

## How to test


- Switch to VS Code Insiders: https://ak-gitpod-walkthrough.staging.gitpod-dev.com/preferences
- Start a workspace.
- On the Welcome screen, look for the new walkthrough on the right side
- Review the walkthrough, click links and verify everything works as expected

## Release Notes

```release-note
VS Code: Add a Get Started with Gitpod walkthrough
```
